component/org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls/src/test/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/mutualtls/MutualTLSClientAuthenticatorTest.java

```diff
@@ -57,8 +58,8 @@
 import java.util.Map;
 
 import static org.mockito.Matchers.any;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.api.mockito.PowerMockito.when;
+import static org.mockito.Matchers.anyString;
+import static org.powermock.api.mockito.PowerMockito.*;
```

Comment:
Shall we add the imports separately?

Reply:
fixed in https://github.com/wso2-extensions/identity-oauth-addons/pull/111/commits/c9c74e14ff756102f1db0080b516e21c2315e296